### PR TITLE
Wider paragraph in alert and update hide to dismiss

### PR DESCRIPTION
**Pull request by chrisrhymes**
Make the paragraph text wider, update 'Hide' to 'Dismiss' so it's different to the Cookie banner. Add tests.

### CODE REVIEW

File: src/library/structure/AlertBanner/AlertBanner.stories.tsx [hunk 1]
```diff
@@ -1,48 +1,78 @@
...
-    uid: "4",
-    alertType: "london_bridge",
-    children: childrenMessage
+  title: 'Something terrible has happened',
+  uid: '4',
+  alertType: 'london_bridge',
+  children: childrenMessage,
+};
+
+export const AlertBannerWithLongText = Template.bind({});
```
Comment (chrisrhymes) [inline src/library/structure/AlertBanner/AlertBanner.stories.tsx:66]: Long text to show the wider paragraph and the spacing between the text and the dismiss button.

File: src/library/structure/AlertBanner/AlertBanner.styles.js [hunk 2]
```diff
@@ -1,88 +1,118 @@
...
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  flex-wrap: nowrap;
+`;
 
 export const BannerContentContainer = styled.div`
-    max-width: calc(100% - 85px);
-`
+  flex-grow: 1;
```
Comment (chrisrhymes) [inline src/library/structure/AlertBanner/AlertBanner.styles.js:70]: Use flexbox to auto adjust the width

File: src/library/structure/AlertBanner/AlertBanner.styles.js [hunk 3]
```diff
@@ -1,88 +1,118 @@
...
-    margin-bottom: 10px;
-`
+  font-weight: bold;
+  margin-bottom: ${(props) => props.theme.theme_vars.spacingSizes.small};
+`;
+
 export const BannerContent = styled.div`
-`
+  p {
+    max-width: 100%;
```
Comment (chrisrhymes) [inline src/library/structure/AlertBanner/AlertBanner.styles.js:81]: Override default paragraph width

File: src/library/structure/AlertBanner/AlertBanner.tsx [hunk 4]
```diff
@@ -1,77 +1,89 @@
...
+            >
+              <CloseIcon
+                colourFill={
+                  alertType === 'warning'
+                    ? themeContext.theme_vars.colours.black
+                    : themeContext.theme_vars.colours.white
+                }
+              />
+              <br />
+              Dismiss
```
Comment (chrisrhymes) [inline src/library/structure/AlertBanner/AlertBanner.tsx:43]: Change text to 'Dismiss' from 'Hide'

**Review by DamienBurkeWN — APPROVED** [review]
Should probably drop those console.log calls but otherwise 👍

**Pull request merged**